Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Custom Columns/TextAndImage.cs

```diff
@@ -26,11 +26,11 @@ public class TextAndImage : IComparable<TextAndImage>
         /// <summary>
         /// The text
         /// </summary>
-        public string Text;
+        public string _text;
```

Comment:
These are public variables, so should be in Capitals.

Reply:
Reverted the changes